llvm/unittests/CodeGen/AArch64SelectionDAGTest.cpp

```diff
@@ -826,7 +826,7 @@ TEST_F(AArch64SelectionDAGTest,
   EXPECT_EQ(KnownAVGFLOORU.One, Ones);
 
   SDValue AVGFLOORS =
-      DAG->getNode(ISD::AVGFLOORU, Loc, Int16Vec8VT, ZextOp0, ZextOp1);
+      DAG->getNode(ISD::AVGFLOORS, Loc, Int16Vec8VT, ZextOp0, ZextOp1);
```

Comment:
I thought I'd fixed this diff in 6acc50ae5b5971af563a0f40b5f0279870171541 ? Please can you rebase